admin-console/src/main/webapp/src/components/Wrappers/BreadCrumbLinkItem.tsx

```diff
@@ -0,0 +1,18 @@
+import { BreadcrumbItem, BreadcrumbItemProps } from '@patternfly/react-core';
+import * as React from 'react';
+
+import { OuiaComponentProps, withoutOuiaProps } from '../../utils/Ouia';
```

Comment:
We can remove the Ouia stuff for now.

Reply:
removing it breaks the link?

Reply:
Yes. You might need to remove the usage of it. Shouldn't impact on the functionality of the component

Reply:
I did remove the usage.. this is what happens...

https://user-images.githubusercontent.com/86322239/159773766-1eeed873-0654-4855-9fb6-663f4de0e66f.mov